### PR TITLE
Fix grammar and spelling

### PR DESCRIPTION
**Pull request by simonua**

### CODE REVIEW

**Comment by simonua** [comment]
@janl, from one fellow German to another, please take a look at my suggested fixes. =P 

Vielen Dank!

File: locales/de.json [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
   "Generate shell code to use npx as the \"command not found\" fallback.": "Erzeuge Shellcode um npx als Alternative zu \"Programm konnte nicht gefunden werden\" zu benutzen.",
-  "Ignores existing binaries in $PATH, or in the local project. This forces npx to do a temporary install and use the latest version.": "Ignoriere bestehende Programme innerhalb von $PATH, oder im lokalen Projekt. Dies zwingt npx dazu die neuste Version herunterzuladen und zu benutzen.",
+  "Ignores existing binaries in $PATH, or in the local project. This forces npx to do a temporary install and use the latest version.": "Ignoriere bestehende Programme innerhalb von $PATH oder im lokalen Projekt. Dies zwingt npx dazu, die neuste Version herunterzuladen und zu benutzen.",
   "npm binary to use for internal operations.": "npm-Programm für die interne Benutzung.",
-  "For the full documentation, see the manual page for npx(1).": "In der Manpage npx(1) ist die gesammte Dokumentation einzusehen.",
+  "For the full documentation, see the manual page for npx(1).": "In der Manpage npx(1) ist die gesamte Dokumentation einzusehen.",
```
Comment (simonua) [inline locales/de.json:12]: _Manpage_ -> _Beschreibung_?

**Review by zkat — APPROVED** [review]
"ok, looks fine" is about all I can say 😂

**Comment by zkat** [comment]
I'm gonna merge this now, and we can make more corrections in the future, since I don't wanna yank in @janl all the time for reviews, and I'd rather just take a PR if I can <3

**Comment by simonua** [comment]
@zkat, thank you. I have a feeling Jan wouldn't object terribly. =P Feel free to ping me when you have locales for German you need, please.

**Comment by zkat** [comment]
@simonua I totally already did 😂 https://github.com/zkat/npx/issues/39#issuecomment-314653017

**Pull request merged**